src/test/resources/application.yml

```diff
@@ -12,3 +12,4 @@ session-couchbase:
     enabled: false
 
 logging.level.com.github.mkopylec.sessioncouchbase: trace
+#logging.level.root: DEBUG
```

Comment:
Oops, sorry about those sloppy things. I was going crazy with the cookie problem on tomcat :-D